www/core/nightlies/next_patch_list.xml

```diff
@@ -1,15 +1,15 @@
+	<extension name="Joomla" element="joomla" type="file" version="4.4.12-dev" targetplatformversion="4.0.4" detailsurl="https://update.joomla.org/core/nightlies/next_patch_extension.xml" />
+	<extension name="Joomla" element="joomla" type="file" version="4.4.12-dev" targetplatformversion="4.0.5" detailsurl="https://update.joomla.org/core/nightlies/next_patch_extension.xml" />
+	<extension name="Joomla" element="joomla" type="file" version="4.4.12-dev" targetplatformversion="4.0.6" detailsurl="https://update.joomla.org/core/nightlies/next_patch_extension.xml" />
 	<extension name="Joomla" element="joomla" type="file" version="4.0.4" targetplatformversion="4.0" detailsurl="https://update.joomla.org/core/nightlies/next_patch_extension.xml" />
-	<extension name="Joomla" element="joomla" type="file" version="4.4.11-dev" targetplatformversion="4.1" detailsurl="https://update.joomla.org/core/nightlies/next_patch_extension.xml" />
-	<extension name="Joomla" element="joomla" type="file" version="4.4.11-dev" targetplatformversion="4.2" detailsurl="https://update.joomla.org/core/nightlies/next_patch_extension.xml" />
-	<extension name="Joomla" element="joomla" type="file" version="4.4.11-dev" targetplatformversion="4.3" detailsurl="https://update.joomla.org/core/nightlies/next_patch_extension.xml" />
-	<extension name="Joomla" element="joomla" type="file" version="5.2.4-dev" targetplatformversion="4.4.11" detailsurl="https://update.joomla.org/core/nightlies/next_patch_extension.xml" />
-	<extension name="Joomla" element="joomla" type="file" version="4.4.11-dev" targetplatformversion="4.4" detailsurl="https://update.joomla.org/core/nightlies/next_patch_extension.xml" />
-	<extension name="Joomla" element="joomla" type="file" version="5.2.4-dev" targetplatformversion="5.0" detailsurl="https://update.joomla.org/core/nightlies/next_patch_extension.xml" />
-	<extension name="Joomla" element="joomla" type="file" version="5.2.4-dev" targetplatformversion="5.1" detailsurl="https://update.joomla.org/core/nightlies/next_patch_extension.xml" />
-	<extension name="Joomla" element="joomla" type="file" version="5.2.4-dev" targetplatformversion="5.2" detailsurl="https://update.joomla.org/core/nightlies/next_patch_extension.xml" />
+	<extension name="Joomla" element="joomla" type="file" version="4.4.12-dev" targetplatformversion="4.1" detailsurl="https://update.joomla.org/core/nightlies/next_patch_extension.xml" />
+	<extension name="Joomla" element="joomla" type="file" version="4.4.12-dev" targetplatformversion="4.2" detailsurl="https://update.joomla.org/core/nightlies/next_patch_extension.xml" />
+	<extension name="Joomla" element="joomla" type="file" version="4.4.12-dev" targetplatformversion="4.3" detailsurl="https://update.joomla.org/core/nightlies/next_patch_extension.xml" />
+	<extension name="Joomla" element="joomla" type="file" version="5.2.5-dev" targetplatformversion="4.4.11" detailsurl="https://update.joomla.org/core/nightlies/next_patch_extension.xml" />
```

Comment:
```suggestion
	<extension name="Joomla" element="joomla" type="file" version="5.2.5-dev" targetplatformversion="4.4.11" detailsurl="https://update.joomla.org/core/nightlies/next_patch_extension.xml" />
```
If we require to be on the latest 4.4 nightly build before updating to 5.2 nightly, we should change the `targetplatformversion="4.4.11"` to `targetplatformversion="4.4.12"`, to it matched the `4.4.12-dev`of the 4.4. nightlies.
If we not require this and allow also to update from 4.4.11 stable to 5.2 nightly, we can keep this line but then should have another one in addition with the previously mentioned change of the targetplatformversion.